exchange/bidder_test.go

```diff
@@ -2109,6 +2111,7 @@ func TestCallRecordTLSHandshakeTime(t *testing.T) {
 	metricsMock := &metrics.MetricsEngineMock{}
 	metricsMock.Mock.On("RecordTLSHandshakeTime", mock.Anything).Return()
 	metricsMock.On("RecordOverheadTime", metrics.PreBidder, mock.Anything).Once()
+	metricsMock.On("RecordOverheadTime", metrics.MakeHTTPRequest, mock.Anything).Once()
```

Comment:
should update `TestNewMetrics` to assert `MakeHTTPRequest` for go-metric

https://github.com/prebid/prebid-server/blob/eb338b6c4c68b778cf103bb7028dd1245ab1aee9/metrics/go_metrics_test.go#L83-L85

Reply:
Addressed in [39ae0fa](https://github.com/prebid/prebid-server/pull/2722/commits/39ae0fae76ec91aa0ceed7b6085d7910159f148c)